interop/test_utils.go

```diff
@@ -638,12 +639,47 @@ func DoUnimplementedMethod(cc *grpc.ClientConn) {
 	}
 }
 
+// DoPickFirstUnary runs multiple RPCs (rpcCount) and checks that all requests
+// are sent to the same backend.
+func DoPickFirstUnary(tc testpb.TestServiceClient, rpcCount int) {
+	pl := ClientNewPayload(testpb.PayloadType_COMPRESSABLE, 1)
+	req := &testpb.SimpleRequest{
+		ResponseType: testpb.PayloadType_COMPRESSABLE,
+		ResponseSize: int32(1),
+		Payload:      pl,
+		FillServerId: true,
+	}
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+	var serverID string
+	for i := 0; i < rpcCount; i++ {
+		resp, err := tc.UnaryCall(ctx, req)
+		if err != nil {
+			grpclog.Fatalf("failed to do UnaryCall: %v", err)
+		}
+		id := resp.ServerId
+		if id == "" {
+			grpclog.Fatalf("got empty server ID")
+		}
+		if i == 0 {
+			serverID = id
+			continue
+		}
+		if serverID != id {
+			grpclog.Fatalf("got different server ids: %q vs %q", serverID, id)
+		}
+	}
+}
+
 type testServer struct {
+	id string
 }
 
 // NewTestServer creates a test server for test service.
 func NewTestServer() testpb.TestServiceServer {
-	return &testServer{}
+	return &testServer{
```

Comment:
It's not strictly necessary that the Go server implement this test case just yet, since there is a special interop server that's configured with the TXT record etc. which the client will run against. But no problem having it - up to you

Reply:
I will revert it for now. It adds a `uuid` dependency to gRPC.